plugins/hls-fourmolu-plugin/hls-fourmolu-plugin.cabal

```diff
@@ -23,7 +23,7 @@ library
   build-depends:
     , base            >=4.12 && <5
     , filepath
-    , fourmolu        ^>=0.3 || ^>=0.4 || ^>= 0.5
+    , fourmolu        ^>=0.3 || ^>=0.4 || ^>= 0.6
```

Comment:
Do we need to disallow 0.5 or should we just add 0.6?

Reply:
0.5 and 0.6 have the same requirements but 0.6 has bug fix, I don't think we have a reason to allow 0.5. https://github.com/fourmolu/fourmolu/blob/master/CHANGELOG.md